docs/server-side-model-downloads-handover.html

```
@@ -0,0 +1,1273 @@
+    at huggingface.co/settings/tokens if they want full revocation.</p>
+  </div>
+</div>
+
+
+<!-- ─────────────────────────────────────────────────────────── -->
+<h2 class="section" id="eligibility">6. Loopback eligibility gate</h2>
+
+<h3>6.1 The rule</h3>
+<pre><code># app/model_downloader/hf_auth/eligibility.py
+
+def is_hf_auth_eligible() -> bool:
```

HTMLHint (1.9.2)

[error] Special characters must be escaped : [ > ].

(spec-char-escape)